README.md

```diff
@@ -84,25 +89,22 @@ Ensure services are running:
 
 ```ShellSession
 $ docker-compose ps
-monocle_api_1       uwsgi --uid guest --gid no ...   Up      0.0.0.0:9876->9876/tcp, 0.0.0.0:9877->9877/tcp
-monocle_crawler_1   monocle --elastic-conn ela ...   Up
-monocle_elastic_1   /usr/local/bin/docker-entr ...   Up      0.0.0.0:9200->9200/tcp, 9300/tcp
-monocle_web_1       docker-entrypoint.sh /bin/ ...   Up      0.0.0.0:8080->8080/tcp
 ```
 
 You might need to check the crawler logs to ensure the crawler started to fetch changes:
 
 ```ShellSession
 $ docker-compose logs -f crawler
+$ docker-compose logs -f crawler-legacy
 ```
 
 You should be able to access the web UI at <http://localhost:8080>.
 
 After a change in the configuration file, the api and crawler services need to be restarted:
 
 ```ShellSession
-$ docker-compose restart api
-$ docker-compose restart crawler
+$ docker-compose restart api-legacy
```

Comment:
docker-compose restart api

The new api still need to be restarted. Better to add it as of now.

Reply:
Let's fix that with https://github.com/change-metrics/monocle/issues/494 instead